README.md

```diff
@@ -1,4 +1,4 @@
-# WordPress Playground
+# WordPress Playground and PHP WASM (WebAssembly)
```

Comment:
This is already a in trunk, I must have squashed this PR to the last trunk commit. It doesn't matter, I'll leave it like this.